packages/css-dev-tools/styleLintConfig.js

```diff
@@ -2,9 +2,19 @@
 
 module.exports = {
   extends: ['stylelint-config-standard'],
-  plugins: ['stylelint-scss'],
+  plugins: ['@mozaic-ds/stylelint-plugin-mozaic', 'stylelint-scss'],
   rules: {
     'at-rule-no-unknown': null,
-    'scss/at-rule-no-unknown': true,
+    'unit-whitelist': ['px', 'em', '%', 'rem', 'ms', 'deg', 'vw'],
```

Comment:
The "vh" unity is missing

Reply:
why this ? we are mostly listing all units here ? Can't we just allow everything ?

Reply:
@gael-boyenval  We surely can just remove the rule to allow everything, it was mostly to test plugin rule with native ones.

Reply:
ok